### PR TITLE
Added getting started page in Highcharts Stock docs.

### PR DESCRIPTION
**Pull request by bm64**
Added getting started page in Highcharts Stock docs.

### CODE REVIEW

File: docs/stock/getting-started-stock.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    
+    <script src="https://code.highcharts.com/stock/highstock.js"></script> 
+
+Load Highcharts Stock as a module when a project needs both Highcharts and Highcharts Stock loaded at the same time. Place the script tag or import statement after loading the main library.
+
+_Example of loading both libraries in a webpage_
+
+    
+    <script src="https://code.highcharts.com/highcharts.js"></script>
+    <script src="https://code.highcharts.com/stock/highstock.js"></script>
```
Comment (pawelfus) [inline docs/stock/getting-started-stock.md:24]: `highstock.js` -> `modules/stock.js`

**Review by pawelfus — CHANGES_REQUESTED** [review]
One tiny thing, otherwise good!

**Review by pawelfus — APPROVED** [review]

**Comment by pawelfus** [comment]
A word of explanation:
All other products have "getting started" doc-page. This one is almost the same as the one for the Gantt product

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**